src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/Config/RemoveAzureRmContainerServiceAgentPoolProfileCommand.cs

```diff
@@ -42,6 +42,7 @@ public class RemoveAzureRmContainerServiceAgentPoolProfileCommand : Microsoft.Az
         [Parameter(
             Mandatory = true,
             Position = 1,
+            ParameterSetName = "NameParameterSet",
```

Comment:
I was just wondering why it was added. As long as the scenario where only the `ContainerService` parameter is provided is covered, this will be fine